src/BloomBrowserUI/bookEdit/js/bloomImages.js

```diff
@@ -54,38 +54,62 @@ function SetupImage(image) {
     });
 }
 
+function SetButtonModifier(container) {
+    var buttonModifier = '';
+    var $container = $(container);
+    if ($container.height() < 95 || $container.width() < 172) {
```

Comment:
Can you give these numbers meaningful names or relate them to any other dimensions? Where did these particular sizes come from?